app/controllers/course/assessment/question/programming_controller.rb

```diff
@@ -44,8 +44,8 @@ def edit
   end
 
   def update
-    duplicate_and_replace_programming_question
     result = @programming_question.class.transaction do
+      duplicate_and_replace_programming_question
```

Comment:
If you're rearranging items, please squash the commit with the respective commit?

Reply:
Done